package.json

```diff
@@ -10,6 +10,7 @@
     "@patternfly/patternfly": "^2.31.0",
     "@patternfly/react-core": "3.153.13",
     "@patternfly/react-icons": "3.15.16",
+    "@patternfly/react-styles": "^3.7.13",
```

Comment:
@jkozol, could you please remove `^`? The cockpit-ci has `bots` to upgrade dependencies automatically, but it'll not update package with version start with `^`. Thanks!

Reply:
Thanks! I fixed this.